src/MonitorProfiler/Logging/StdErrLogger.h

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#pragma once
+
+#include <memory>
+#include <vector>
+#include "Logger.h"
+#include "../Environment/Environment.h"
+
+/// <summary>
+/// Logs messages to the stderr stream.
+/// </summary>
+class StdErrLogger final :
+    public ILogger
+{
+private:
+    const static LogLevel DefaultLevel = LogLevel::None;
+    const static int MaxEntrySize = 1000;
```

Comment:
It seems like in some places `MaxEntrySize` is `size_t` and others it's `int`. Is there a reason for this, and if so, is it worth documenting?

Reply:
This should be changed to `size_t` to match the method signatures. I'll fix this one.